miri.bat

```diff
@@ -0,0 +1,8 @@
+:: This makes execution of ./miri on Linux and Windows the same.
+:: Windows will not executable the bash script, and select this.
```

Comment:
```suggestion
:: Windows will not execute the bash script, and select this.
```
Does this depend on the shell ones uses (old cmd / powershell / not sure if there's anything else)?

Reply:
No it does not matter

Reply:
Unless they modify their PATHEXT env var themselves, which is on them.